### PR TITLE
Survey Ranking template for between 1-5

### PR DESCRIPTION
**Pull request by Koji23**
Screen Shot 2022-07-05 at 10 28 29 AM

### CODE REVIEW

**Review by gardenFiend138 — COMMENTED** [review]
are we intending on adding a readme for each of these templates? noticed the blank readme file

also can we add Charles and Olive as reviewers to this and check how this does across devices/ landscape/ etc?

**Comment by Koji23** [comment]
> are we intending on adding a readme for each of these templates? noticed the blank readme file
> 
> also can we add Charles and Olive as reviewers to this and check how this does across devices/ landscape/ etc?

yes I think we can aim to ship readme's for each template individually to detail how they should work.

File: ranking_survey/index.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M7.80309 1.14768C8.06564 0.885137 8.06564 0.459453 7.80309 0.196909C7.54055 -0.0656362 7.11486 -0.0656362 6.85232 0.196909L4 3.04923L1.14768 0.196909C0.885137 -0.0656362 0.459453 -0.0656362 0.196909 0.196909C-0.0656362 0.459453 -0.0656362 0.885137 0.196909 1.14768L3.04923 4L0.196909 6.85232C-0.0656362 7.11486 -0.0656362 7.54055 0.196909 7.80309C0.459453 8.06564 0.885137 8.06564 1.14768 7.80309L4 4.95077L6.85232 7.80309C7.11486 8.06564 7.54055 8.06564 7.80309 7.80309C8.06564 7.54055 8.06564 7.11486 7.80309 6.85232L4.95077 4L7.80309 1.14768Z"
+            fill="#111111"
+          />
+        </svg>
+      </div>
+      <h1>Hi, Olivia</h1>
```
Comment (gardenFiend138) [inline ranking_survey/index.html:132]: maybe a good place to recommend using tag substitution here? `<h1> Hi, {{ name | default: 'There' }}</h1>`?

**Review by gardenFiend138 — APPROVED** [review]

**Review by iAmWillShepherd — APPROVED** [review]
Token approval

**Pull request merged**